### ISSUE TITLE
Ability to filter by author

### ISSUE THREAD

**Issue by daveidivide** [mhutchie/vscode-git-graph#727]
It would be handy to have the ability to filter the list of git history by a specific author.

I work on a project with a lot of other developers and even something I comitted a few days ago can soon be buried with everything else going on.

Ideally I'd like to be able to filter to just see git activity from myself and be able to easily find what I have done.

**Comment by bilogic** [comment]
Yes, this is very useful for trunk based development where we don't have many branches.

**Comment by jnnnnn** [comment]
duplicate of #171

**Comment by huaiguoguo** [comment]
two years
